gcp/table_gcp_tag_binding.go

```diff
@@ -0,0 +1,126 @@
+		Name:        "gcp_tag_binding",
+		Description: "GCP Tag Binding",
+		List: &plugin.ListConfig{
+			Hydrate:           listGcpTagBindings,
+			KeyColumns:        plugin.SingleColumn("parent"),
+			ShouldIgnoreError: isIgnorableError([]string{"InvalidArgument"}),
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name of the TagBinding. This is a string of the form: `tagBindings/{full-resource-name}/{tag-value-name}`.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "parent",
+				Description: "The full resource name of the resource the TagValue is bound to.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "tag_value",
+				Description: "The TagValue of the TagBinding. Must be of the form `tagValues/456`.",
+				Type:        proto.ColumnType_STRING,
+			},
+			// Additional fields as needed
+			// Standard Steampipe columns
+			{
+				Name:        "title",
+				Description: "Title of the resource.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+			},
+			{
+				Name:        "akas",
+				Description: ColumnDescriptionAkas,
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     plugin.HydrateFunc(getTagBindingAka),
+				Transform:   transform.FromValue(),
+			},
+			//Standard GCP columns
+			{
+				Name:        "project",
+				Description: ColumnDescriptionProject,
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     plugin.HydrateFunc(getProject).WithCache(),
+				Transform:   transform.FromValue(),
+			},
+		},
+	}
+}
+
+//// LIST FUNCTION
+
+func listGcpTagBindings(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	parent := d.EqualsQuals["parent"].GetStringValue()
+
+	if parent == "" {
+		return nil, nil
+	}
+
+	pageSize := types.Int64(300)
+
+	limit := d.QueryContext.Limit
+	if d.QueryContext.Limit != nil {
+		if *limit < *pageSize {
+			pageSize = limit
+		}
+	}
+
+	// Create Service Connection
+	client, err := resourcemanager.NewTagBindingsClient(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	// Construct the request
+	req := &resourcemanagerpb.ListTagBindingsRequest{
+		Parent:   parent,
+		PageSize: int32(*pageSize),
+	}
+
+	// Iterate through the results
+	it := client.ListTagBindings(ctx, req)
+	for {
+		tagBinding, err := it.Next()
+		if err == iterator.Done {
+			break
+		}
+		if err != nil {
+			return nil, err
+		}
+
+		d.StreamListItem(ctx, tagBinding)
+
+		// Context cancellation check
+		if d.RowsRemaining(ctx) == 0 {
+			break
+		}
+	}
+
+	return nil, nil
+}
+
+//// HYDRATE FUNCTIONS
+
+func getTagBindingAka(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	data := h.Item.(*resourcemanagerpb.TagBinding)
+	region := "global"
+	akas := []string{"gcp://cloudresourcemanager.googleapis.com/projects/" + data.Name + "/tagBindings/" + data.Name + "/tagValues/" + data.Name + "/regions/" + region}
```

Comment:
@Priyanka-Chatterjee-2000 The format of `Aka` is not correct, can you please check again?
```
> select akas From gcp_tag_binding where parent = '//cloudresourcemanager.googleapis.com/projects/111111111111'
[
 {
  "akas": [
   "gcp://cloudresourcemanager.googleapis.com/projects/tagBindings/%2F%2Fcloudresourcemanager.googleapis.com%2Fprojects%2F111111111111/tagValues/222222222222/tagBindings/tagBindings/%2F%2Fcloudresourcemanager.googleapis.com%2Fprojects%2F111111111111/tagValues/222222222222/tagValues/tagBindings/%2F%2Fcloudresourcemanager.googleapis.com%2Fprojects%2F111111111111/tagValues/222222222222/regions/global"
  ]
 }
]

```